style.css

```diff
@@ -1283,4 +3048,71 @@ body {
     width: 20px;
     height: 20px;
   }
+
+  .second-page {
+    min-height: 573px;
+  }
+
+  .second-page__img {
+    margin-bottom: 3%;
+  }
+
+  .second__wrap {
+    margin-bottom: 12px;
+  }
+
+  .third-page {
+    background: linear-gradient(to right, #EBE3D8 98px, #000 98px, #000 137px, #EBE3D8 137px);
```

Comment:
Codacy found an issue: [Expected "#EBE3D8" to be "#ebe3d8" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/Flack_project/pullRequest?prid=6358146)